### PR TITLE
Create abuse_quickbooks_impersonation_intuit.yml

### PR DESCRIPTION
**Pull request by zoomequipd**
# Description

Add coverage for observed Quickbooks Abuse

# Associated samples

- [Write Up and Sample](https://www.notion.so/sublimesecurity/EXTERNAL-Invoice-INTUITS46-eml-12a04655fc9d80c2bef4c600498d5191?pvs=4)

### CODE REVIEW

**Comment by zoomequipd** [comment]
No shared samples, looks low volume.  marking for review

File: detection-rules/abuse_quickbooks_impersonation_intuit.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: "Brand Impersonation: QuickBooks Notification From Inuit Themed Company Name"
```
Comment (aidenmitchell) [inline detection-rules/abuse_quickbooks_impersonation_intuit.yml:1]:
```suggestion
name: "Brand Impersonation: QuickBooks Notification From Intuit Themed Company Name"
```

File: detection-rules/abuse_quickbooks_impersonation_intuit.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+description: "This detection rule matches on QuickBooks notifications that feature company names impersonating Inuit and Quickbooks"
```
Comment (aidenmitchell) [inline detection-rules/abuse_quickbooks_impersonation_intuit.yml:2]:
```suggestion
description: "This detection rule matches on QuickBooks notifications that feature company names impersonating Intuit and QuickBooks."
```

File: detection-rules/abuse_quickbooks_impersonation_intuit.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    or any(headers.reply_to,
+           (
+             strings.icontains(.email.email, 'intuit')
+             or strings.icontains(.email.domain.domain, 'quickbooks')
+           )
+           and not (.email.domain.root_domain in ('intuit.com', 'quickbooks.com'))
+    )
+    // the "company" part of the message
+    or regex.icontains(body.html.raw,
+                       '<p class="companyName info-line" style="margin: 0; line-height: 1.5em; font-weight: 600;">.*(?:Inuit|Quickbooks).*</p></div>'
```
Comment (aidenmitchell) [inline detection-rules/abuse_quickbooks_impersonation_intuit.yml:29]:
```suggestion
                       '<p class="companyName info-line" style="margin: 0; line-height: 1.5em; font-weight: 600;">.*(?:Intuit|Quickbooks).*</p></div>'
```

**Review by aidenmitchell — CHANGES_REQUESTED** [review]

**Review by aidenmitchell — APPROVED** [review]

**Pull request merged**